packages/multi-provider/tests/index.test.ts

```diff
@@ -25,78 +22,82 @@ describe('multi-provider', async () => {
+  beforeAll(() => {
     mp = new MultiProvider<Domain>();
     mp.registerDomain(chainADomain);
     mp.registerDomain(chainBDomain);
   });
 
   it('returns an array of domains', () => {
     const values = mp.registeredDomains;
-    expect(values).to.include(chainADomain);
-    expect(values).to.include(chainBDomain);
+    expect(values).toContainEqual(chainADomain);
+    expect(values).toContainEqual(chainBDomain);
   });
 
   it('returns array of domain numbers', () => {
     const numbers = mp.domainNumbers;
-    expect(numbers).to.include(chainADomain.domain);
-    expect(numbers).to.include(chainBDomain.domain);
+    expect(numbers).toContain(chainADomain.domain);
+    expect(numbers).toContain(chainBDomain.domain);
   });
 
   it('returns an array of domain names', () => {
     const names = mp.domainNames;
-    expect(names).to.include(chainADomain.name);
-    expect(names).to.include(chainBDomain.name);
+    expect(names).toContain(chainADomain.name);
+    expect(names).toContain(chainBDomain.name);
   });
 
   it('returns an array of missing providers', () => {
     const missing = mp.missingProviders;
     const aRegistered = mp.providers.has('a');
     const bRegistered = mp.providers.has('b');
-    expect(aRegistered).to.be.false;
-    expect(bRegistered).to.be.false;
-    expect(missing).to.include(chainADomain.name);
-    expect(missing).to.include(chainBDomain.name);
+    expect(aRegistered).toEqual(false);
+    expect(bRegistered).toEqual(false);
+    expect(missing).toContain(chainADomain.name);
+    expect(missing).toContain(chainBDomain.name);
   });
 
   it('returns domain for given nameOrDomain', () => {
     const domainFromName = mp.resolveDomain(chainADomain.name);
     const domainFromDomain = mp.resolveDomain(chainADomain.domain);
-    expect(domainFromName).to.equal(domainFromDomain).to.equal(1000);
+    expect(domainFromName).toEqual(domainFromDomain);
+    expect(domainFromName).toEqual(1000);
+    expect(domainFromDomain).toEqual(1000);
   });
 
   it('returns name for given nameOrDomain', () => {
     const nameFromName = mp.resolveDomainName(chainADomain.name);
     const nameFromDomain = mp.resolveDomainName(chainADomain.domain);
-    expect(nameFromName).to.equal(nameFromDomain).to.equal('a');
+    expect(nameFromName).toEqual(nameFromDomain);
+    expect(nameFromName).toEqual('a');
+    expect(nameFromDomain).toEqual('a');
   });
 
   it('resolveDomainName errors if domain is not found', () => {
-    expect(() => mp.resolveDomainName(4000)).to.throw();
-    expect(() => mp.resolveDomainName('hi')).to.throw();
+    expect(() => mp.resolveDomainName(4000)).toThrow();
+    expect(() => mp.resolveDomainName('hi')).toThrow();
   });
 
   it('returns whether a given domain is registered', () => {
     const known = mp.knownDomain('a');
     const unknown = mp.knownDomain('c');
-    expect(known).to.be.true;
-    expect(unknown).to.be.false;
+    expect(known).toEqual(true);
+    expect(unknown).toEqual(false);
```

Comment:
Isn't there a toBeTrue method? And toBeFalse?

Reply:
No, unfortunately jest doesn't have `toBeTrue` or `toBeFalse` methods. There's `toBeTruthy()` and `toBeFalsey()` but it works similar to how an if statement would evaluate a value to a boolean.